task/07-yield-tasks.js

```diff
@@ -82,7 +104,15 @@ function* getFibonacciSequence() {
  *
  */
 function* depthTraversalTree(root) {
-    throw new Error('Not implemented');
+    let stack = [root];
+    while (stack.length > 0) {
+        root = stack.pop();
+        yield root;
+        if (typeof root.children !== 'undefined')
+            for (let value of root.children.reverse())
```

Comment:
use {} when the blok statement is on a new line. Check for this in other methods as well